src/main/java/mate/academy/model/ShoppingCart.java

```diff
@@ -0,0 +1,60 @@
+package mate.academy.model;
+
+import java.util.List;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.MapsId;
+import javax.persistence.OneToMany;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "shopping_carts")
+public class ShoppingCart {
+    @Id
+    private Long id;
+    @OneToMany
+    @JoinTable(name = "shopping_carts_tickets",
+            joinColumns = @JoinColumn(name = "shopping_carts_id"),
+            inverseJoinColumns = @JoinColumn(name = "tickets_id"))
+    private List<Ticket> tickets;
+    @OneToOne(fetch = FetchType.LAZY)
```

Comment:
```suggestion
    @OneToOne
```
Why do you need a fetch type lazy? Eager fetch type is not an issue in One to One)